openweathermap-ios/openweathermap-ios/Maps/ViewControllers/MapViewController.swift

```diff
@@ -85,6 +101,7 @@ extension MapViewController: MKMapViewDelegate {
         let annotationView = AnnotationView(annotation: annotation, reuseIdentifier: "annotation")
         annotationView.canShowCallout = true
         annotationView.isSelected = true
+        annotationView.rightCalloutAccessoryView = UIButton(type: .detailDisclosure)
```

Comment:
Are you using this button? Have you connected some IBAction?

Reply:
Now it has been added to keep pin information always visible.
But in the future it will be used as a button.
The two images exemplify with the button and without the button.

With
![Depois](https://user-images.githubusercontent.com/21261476/79175102-23c66f00-7dd3-11ea-890b-c4236249a216.png)

Without
![Antes](https://user-images.githubusercontent.com/21261476/79175118-304ac780-7dd3-11ea-95fa-5566c05906cc.png)

Without having to click on the pin to display the information.
I didn't know how to solve it any other way :(.
It was the way I managed to resolve it temporarily.

Reply:
Ok, it isn't a big deal.